application/src/main/java/com/knubisoft/application/entity/EmailDetails.java

```diff
@@ -0,0 +1,15 @@
+package com.knubisoft.application.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmailDetails {
+    private String recipient;
+    private String msgBody;
+    private String subject;
+    private String attachment;
```

Comment:
Do you want to implement List of attachments and restrict it by 3 files per message, something like that?